semantic_router/layer.py

```diff
@@ -391,6 +403,18 @@ def _retrieve(self, xq: Any, top_k: int = 5) -> List[dict]:
         scores, routes = self.index.query(vector=xq, top_k=top_k)
         return [{"route": d, "score": s.item()} for d, s in zip(routes, scores)]
 
+    def _set_aggregation_method(self, aggregation: str = "SUM"):
+        if aggregation == "SUM":
+            return lambda x: sum(x)
+        elif aggregation == "MEAN":
+            return lambda x: np.mean(x)
+        elif aggregation == "MAX":
+            return lambda x: max(x)
```

Comment:
nit: let's please use lowercase for the `aggregation` here to keep in line with [Pandas .agg function names](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.agg.html)

Reply:
I can change that now. I just followed TensorFlow's convention instead.

Reply:
I have changed it now.